src/AutoRest.CSharp/Common/Generation/Writers/CodeWriter.cs

```diff
@@ -175,6 +176,9 @@ public CodeWriter Append(FormattableString formattableString)
                     case CodeWriterDeclaration declaration:
                         Append(declaration);
                         break;
+                    case ValueExpression expression:
+                        this.WriteValueExpression(expression);
```

Comment:
the `WriteValueExpression` is an extension method of `CodeWriter` therefore we have to have the `this.` here.

I believe we could consider to change it to an instance method - it just is as important as the other methods of writing things since we are using expressions more and more
But we could put it into a separate PR, this PR is complicated enough

Reply:
OK, I see